include/nuttx/sched.h

```diff
@@ -134,6 +134,7 @@
 #define TCB_FLAG_FORCED_CANCEL     (1 << 13)                     /* Bit 13: Pthread cancel is forced */
 #define TCB_FLAG_JOIN_COMPLETED    (1 << 14)                     /* Bit 14: Pthread join completed */
 #define TCB_FLAG_FREE_TCB          (1 << 15)                     /* Bit 15: Free tcb after exit */
+#define TCB_FLAG_SIGDELIVER        (1 << 16)                     /* Bit 16: Deliver pending signals */
```

Comment:
```suggestion
#define TCB_FLAG_SIGDELIVER        (1 << 2)                      /* Bit 2: Deliver pending signals */
```
move before line 120

Reply:
Currently, bit 2 is not used. I don't know if there are other reasons why bit 2 was not used before. If there is no other impact, wouldn't it be better to arrange it more compactly, such as letting TCB_FLAG_POLICY use bits 2-3, TCB_FLAG_CPU_LOCKED use bit 4, and so on? I wonder if this is okay?

Reply:
Can the "scheduling policy" change to the placeholder to 2-3 bits? And then arrange the rest in order. I feel it's easier to understand that way.

Reply:
either is fine to me

Reply:
Done.